ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,95 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  intermediate411:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.11"
+  latest:
```

Comment:
Seem like this release image is never used.

Reply:
I am not using release:latest but if I remove it from the config then I get a validation error when running make jobs.

Reply:
I guess ARO is using some stable build, though we can not determinate which version to be sued in ARO install, to be closer to the source version, I suggest to point release:latest to 4.10 stable build instead of nightly build.

Reply:
[dedf5c0](https://github.com/openshift/release/pull/36814/commits/dedf5c0f78cb6c55f3df45f3da1526fa0889b92c)